src/python_testing/TC_TSTAT_4_2.py

```diff
@@ -647,6 +648,29 @@ async def test_TC_TSTAT_4_2(self):
         self.step("18")
         if self.pics_guard(self.check_pics("TSTAT.S.F08") and self.check_pics("TSTAT.S.A0050") and self.check_pics("TSTAT.S.Cfe.Rsp")):
 
+            # Generate list of tuples of scenarios and number of remaining presets per scenario allowed
```

Comment:
It's a list of lists, not a list of tuples....  Will make some code suggestions to actually make it tuples and maybe improve readability.

Reply:
Took the tuple suggestion, and learned something new about Python syntax. Thanks!